mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
+<% unless version == 'ga' -%>
```

Comment:
goimports should figure the right thing out w/o this, now that we're in this file - I don't think we need these lines as hints. We've got a third provider variant (alpha / eap) kicking around in a different namespace (and incorrect hints *won't* get figured out right)

Reply:
Long night, eh?

Fixed